### DISCUSSION TITLE
How to save PDF Multipage certificate while using Ventoy

### DISCUSSION THREAD

**Discussion by davidlahoz** [PartialVolume/shredos.x86_64#237] [Q&A]
Hello,

I am currently using ShredOS last release release `shredos-2023.08.2_25.0_x86-64_0.35_20240207.iso` and running it using Ventoy. I have a question regarding the "erase certificates".

Is there a way to save certificates directly on the USB drive that I am using to run ShredOS? I would like to ensure that my certificates are stored in the USB drive to later one save them in another location. I asume that if I create a bootable with Rufus or Etcher it would save the files, but I did not find a way to store it in the USB drive whiel using Ventoy.

Any guidance or suggestions would be greatly appreciated.

Thank you.

**Comment by PartialVolume** [comment]
The next release, which will hopefully be published with pre built .iso and .img files in April, will automatically save to Ventoy USB drives.

**Comment by davidlahoz** [comment]
Thanks @PartialVolume
